src/main/java/cloud/eppo/BaseEppoClient.java

```diff
@@ -122,6 +125,43 @@ protected void loadConfiguration() {
     }
   }
 
+  private Timer pollTimer;
+
+  protected void stopPolling() {
+    if (pollTimer != null) {
+      pollTimer.cancel();
+    }
+  }
+
+  protected void startPolling() {
+    startPolling(DEFAULT_POLLING_INTERVAL_MILLIS);
+  }
+
+  void startPolling(long pollingIntervalMs) {
```

Comment:
is this package protected for a reason or should it just be "normal" `protected`?

Reply:
fixed